html/includes/graphs/device/saf_radioTxPower.inc.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+require 'includes/graphs/common.inc.php';
+
+$rrdfilename = $config['rrd_dir'].'/'.$device['hostname'].'/saf.rrd';
+
+if (file_exists($rrdfilename)) {
```

Comment:
Need to use rrdtool_check_rrd_exists() to check for rrd files - rrdcached based ones may not have local files